README.adoc

```diff
@@ -77,12 +77,12 @@ NAME               HOST/PORT
 tekton-dashboard   tekton-dashboard-tekton-pipelines.apps.kabanero.os.fyre.ibm.com          tekton-dashboard   <all>   
 ----
 +
-. From Tekton Dashboard, go to **Webhooks** and click on the `+` next to **Add Webhook**.
+. From the pipelines dashboard, go to **Webhooks** and click on the `+` next to **Add Webhook**.
 . From **Webhook Settings**, give your webhook a name, repository URL, and access token. The webhook name can be anything. This example uses a Java MicroProfile pipeline, so your codebase should be Java MicroProfile-based. If you don't have a Java MicroProfile codebase, you can fork link:https://github.com/dewan-ahmed/java-microprofile[this repo]. To learn how to create a personal access token for GitHub, see link:https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line[GitHub's documentation].
```

Comment:
ok, why would we reference this?  https://github.com/dewan-ahmed/java-microprofile[this repo] ?  This isn't hosted at a known open source project.